wurst/util/MapBounds.wurst

```diff
@@ -39,7 +39,6 @@ public function vec2.inBounds() returns boolean
 public function vec3.inBounds() returns boolean
 	return inBoundsXY(this.x, this.y)
 
-
```

Comment:
what?
e: I mean, this edit isn't necessary at all, so this file will appear in the list of edited files for no reason.

Reply:
yeah should be fixed

Reply:
Resolved? You remove an extra line here, yet in your packages, like UnitUtils, there are many more empty lines?